event_metadata/models.py

```diff
@@ -0,0 +1,58 @@
+from pickletools import optimize
+from re import S
+from django.db import models
+
+from accounts.models import *
+from fsm.models import Event
+import PIL
+
+class SimpleTeam(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField(null=True, blank=True)
+
+class Role(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField(null=True, blank=True)
+
+def event_image_upload_path(instance, filename):
+    return f'events/{instance.event.pk}/{filename}/'
+
+class StaffInfo(models.Model):
+    account = models.ForeignKey('accounts.User', on_delete=models.PROTECT, related_name='staff_infos')
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name='staff_infos')
+    description = models.TextField(null=True, blank=True)
+    team = models.ManyToManyField(SimpleTeam, blank=True)
+    role = models.ForeignKey(Role, null=True, blank=True, on_delete=models.PROTECT, related_name='staff_infos')
+    image = models.ImageField(upload_to=event_image_upload_path, blank=True, null=True)
+
+    class Meta:
+        unique_together = ['account', 'event']
+
+    def save(self):
+        super().save()
+        img = PIL.Image.open(self.image.path)
+        img.save(self.image.path, optimize=True, quality=85)  

```

Comment:
I think its not necessary to reduce staff picture quality.
Its good to have a limit on picture size